webapp/src/main/webapp/openmrs.js

```diff
@@ -626,11 +626,17 @@ function hideError(errorName) {
  * @param object(Required) to be limited.
  * @param maxLength(Required) the length of the limit.
  */
-function forceMaxLength(object, maxLength) {
-    if( object.value.length >= maxLength) {
-       object.value = object.value.substring(0, maxLength); 
-    }
-}
+ 
+function forceMaxLength(object, errorFieldId, maxLength) {
```

Comment:
From the ticket summary, is it mentioned that this should be a general purpose function that will be called from anywhere, that is why you can't leave code like that behind